src/content/posts/2025-pyladiescon-cfp-launch.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: "PyLadiesCon 2025: Call for Proposals is Now Open!"
+meta_title: "PyLadiesCon 2025: Call for Proposals is Now Open!"
+description: "PyLadiesCon 2025 is accepting proposals"
+date: 2025-07-11T05:00:00Z
+image: " "
```

Comment:
Looks like this blank image is causing issue with the build. Could you add the image or put some placeholder image?
Or try removing the line completely.

Reply:
I added a placeholder image. Thank you Mariatta.

Reply:
I think you forgot to commit the file.